### DISCUSSION TITLE
How about "sqlkata"?

### DISCUSSION THREAD

**Discussion by won5572** [PiranhaCMS/piranha.core#1391] [General] [answered]
EF is good, but I like it when I used sqlkata.
What do you think about applying sqlkata to Piranha CMS?

sqlkata >> https://sqlkata.com

**Comment by tidyui** [comment]
As we're a fairly small core team we only intend to provide packages for Entity Framework Core. However, the entire repository layer is pluggable, so you could exchange it for anything, like storing data on disk with `JSON` files, or using SqlKata. The only downside with this is of course that you will need to keep track of the change log for upcoming versions and keep your repository layer in sync!

Best regards

**Comment by won5572** [comment, marked as answer]
I got it. I thought too. Thank you for your reply. Have a nice day!

**Reply by tidyui** [reply]
Please consider marking the above as "answer" if you think it answered your question, this helps filtering the discussions :)

Best regards
